src/Raven.Studio/typescript/commands/licensing/getConnectivityToLicenseServerCommand.ts

```diff
@@ -0,0 +1,16 @@
+﻿import commandBase = require("commands/commandBase");
+import endpoints = require("endpoints");
+
+class getConnectivityToLicenseServerCommand extends commandBase {
+ 
+    execute(): JQueryPromise<Raven.Server.Web.Studio.LicenseHandler.ConnectivityToLicenseServer> {
+        const url = endpoints.global.license.licenseServerConnectivity;
+
+        return this.query<any>(url, null)
+            .fail((response: JQueryXHR) => {
+                this.reportError("Failed to get connectivity to license server", response.responseText, response.statusText);
```

Comment:
Failed to connect to license server?

Reply:
done
`Failed to connect the license server`